.github/workflows/release.yml

```
@@ -0,0 +1,151 @@
+name: release
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+  # pull_request:
+  #   branches: [ main ]
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Install maturin
+        run: pip install maturin
+      - name: Build wheels - x86_64
+        run: |
+          maturin build -m roapi-http/Cargo.toml -b bin --target x86_64-apple-darwin --release --out dist
+          pip install roapi-http --no-index --find-links dist --force-reinstall
+      - name: Build wheels - universal2
+        env:
+          DEVELOPER_DIR: /Applications/Xcode.app/Contents/Developer
+          MACOSX_DEPLOYMENT_TARGET: '10.9'
+        run: |
+          # set SDKROOT for C dependencies
+          export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+          maturin build -m roapi-http/Cargo.toml -b bin --release --universal2 --out dist --no-sdist
+          pip install roapi-http --no-index --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+      - name: Archive binary
+        run: |
+          cd target/release
+          tar czvf roapi-http.tar.gz roapi-http
+          cd -
```

I think you can probably simplify these tar commands to:

```suggestion
        run:  tar czvf target/release/roapi-http.tar.gz -C target/release roapi-http
```